src/anchor/anchor.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-anchor($status: primary, $underline: 'false') {
+  --_iui-anchor-popout-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z' /></svg>");
+
+  @include iui-anchor-status($status);
+  border-radius: $iui-border-radius;
+  text-underline-offset: 1px;
+  forced-color-adjust: none;
+  cursor: pointer;
+
+  @if $underline == 'false' {
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  } @else {
+    &:hover {
+      text-decoration: none;
+    }
+  }
+
+  &-blank::after {
+    content: '';
+    display: inline-block;
+    width: 1.5ch;
+    height: 1.5ch;
```

Comment:
This icon does not look vertically centered to me.

Reply:
The height of the `<a>` appears to be 19px & the icon is 12px.  I played around with adding a `margin-top` & `padding-top` to `::after` but wasn't happy with the results yet.  It is using `display: flex` & `align-items: center` so it is trying to center it.

![Screen Shot 2022-03-29 at 12 30 09 PM](https://user-images.githubusercontent.com/849817/160660699-274de5d9-306b-44f9-963b-60a62fdb6c3f.png)

Reply:
maybe use baseline?

Reply:
You can use values like `1px` with vertical-align. Perhaps `vertical-align: -0.1ch;` is what you're looking for?

Reply:
`vertical-align: -0.1ch;` does seem to do the trick!  Tested on different font sizes and visually looks good.

Reply:
Great!